### PR TITLE
Feature/tsd 3188

### PR DESCRIPTION
**Pull request by DanMad**
[TSD-3188], [TSD-3189]

[TSD-3188]: https://opencolleges.jira.com/browse/TSD-3188
[TSD-3189]: https://opencolleges.jira.com/browse/TSD-3189

### CODE REVIEW

File: lib/badge/badge.tsx [hunk 1]
```diff
@@ -2,13 +2,52 @@ import React from 'react';
...
+ * <Badge />
+ *
+ * <!-- Output -->
+ * <span class="uds-badge" unselectable="on">0</span>
+ *
+ * <!-- Declaration -->
+ * <Badge modifiers="success" value={100} className="pad-1-rem" />
+ *
+ * <!-- Output -->
+ * <span class="uds-badge uds-badge--success pad-1-rem" unselectable="on">99+</span>
```
Comment (Narasim-OC) [inline lib/badge/badge.tsx]: Hi @DanMad , You are passing the value 100 but the output is showing 99+ Could you please clarify this.

File: lib/badge/badge.tsx [hunk 2]
```diff
@@ -2,13 +2,52 @@ import React from 'react';
...
+ *
+ *  <!-- Declaration -->
+ * <Badge modifiers="error" value={12} style={{ zIndex: '9999' }}/>
+ *
+ * <!-- Output -->
+ * <span class="uds-badge uds-badge--error" style="z-index:9999" unselectable="on">9+</span>
```
Comment (Narasim-OC) [inline lib/badge/badge.tsx]: HI @DanMad , Same as above. the value you are passing is 12. But the output is showing 9+

File: lib/badge/badge.tsx (hunk 1 above)
Comment (DanMad) [reply lib/badge/badge.tsx]: Yes. This is correct. It's a separate utility handling this, that is called within `<Badge />`. At some stage, I will also document different utilities associated with each component.

File: lib/badge/badge.tsx (hunk 2 above)
Comment (DanMad) [reply lib/badge/badge.tsx]: Same explanation as above ;)

**Pull request merged**